vue.config.js

```diff
@@ -25,22 +25,15 @@ module.exports = {
       .plugin('favicons')
       .use(FaviconsWebpackPlugin, [{
         logo: path.resolve(__dirname, 'src/assets/favicon.svg'),
-        mode: 'webapp', // optional can be 'webapp' or 'light' - 'webapp' by default
-        devMode: 'webapp', // optional can be 'webapp' or 'light' - 'light' by default
```

Comment:
The default values should be fine. I don't think that it is necessary to generate `2732x2048` images on each dev run.